subsys/bluetooth/audio/pacs.c

```diff
@@ -22,6 +22,7 @@
 #include <zephyr/bluetooth/audio/audio.h>
 #include <zephyr/bluetooth/audio/pacs.h>
 #include "../host/conn_internal.h"
+#include <stdint.h>
```

Comment:
Why do we need this? 

We could consider including the more generic `zephyr/types.h` header files (`
#include <zephyr/types.h>`) if we really need anything that isn't already included.

Reply:
We don't need it - removed.